src/ui_sign_bagl.c

```diff
@@ -1,3 +1,24 @@
+/* Tezos Ledger application - Delegate BAGL UI handling
+
+   Copyright 2024 TriliTech <contact@trili.tech>
+   Copyright 2024 Functori <contact@functori.com>
+   Copyright 2023 Ledger
```

Comment:
Why Ledger is being added to copyright?  Did they write code in this file in 2023 sometime?

Reply:
Their last commit was : https://github.com/trilitech/ledger-app-tezos-baking/commit/7a6e463dbf2440d113639826c323a6daf34102ba